src/zenml/models/v2/base/filter.py

```diff
@@ -768,9 +758,6 @@ def generate_custom_query_conditions_for_column(
         filter_ = FilterGenerator(table).define_filter(
             column=column, value=value, operator=operator
         )
-        if isinstance(filter_, StrFilter):
-            filter_.json_encode_value = json_encode_value
```

Comment:
That value seems unused now?

Reply:
Yes, I forgot to remove it from the signature. Fixed it now.